CONTRIBUTING.md

```diff
@@ -293,20 +293,58 @@ python setup.py build_sphinx
 
 ### Flask server
 
-Make sure your machine meets the [OS dependencies](https://superset.incubator.apache.org/installation.html#os-dependencies) before following these steps.
+#### OS Dependencies
+
+Make sure your machine meets the [OS dependencies](https://superset.incubator.apache.org/installation.html#os-dependencies) before following these steps. 
+
+In addition you will need to install the mysql client to get mysql_config, which is required by mysqlclient in requirements-dev.txt.
+
+For Debian/Ubuntu:
+
+```bash
+sudo apt-get install -y libmysqlclient-dev
+```
+
+For OS X via brew:
+
+```bash
+brew install mysql-client
+echo 'export PATH="/usr/local/opt/mysql-client/bin:$PATH"' >> ~/.bash_profile
+source ~/.bash_profile
+```
+
+Developers should use a virtualenv. Anaconda Python users should make sure that conda installs the virtualenv package, otherwise conda packages won't play well with packages installed by pip. In addition, Anaconda Python installs its own version of certifi that must be removed minus dependencies for requirements.txt's certifi to install.
```

Comment:
This makes it seem like we recommend Anaconda and adds complexity. Can we just recommend to people to use vanilla `virtualenv` and to not use Anaconda ("if you use Anaconda, you're on your own").

Idk too much about Anaconda, but my feeling is that it's great for the python data science stuff. If Superset was a data science lib, we should make sure it works with conda. But since we're pretty much a standalone app, we should make sure it works on vanilla virtualenv  and probably docker too.

Reply:
+1 @mistercrunch I would prefer to keep the instructions as simple and require the least dependencies possible.

Reply:
k, removing.